edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/ConnectionProvider.cs

```diff
@@ -20,9 +20,9 @@ public ConnectionProvider(IConnectionManager connectionManager, IEdgeHub edgeHub
             this.messageAckTimeout = messageAckTimeout;
         }
 
-        public Task<IDeviceListener> GetDeviceListenerAsync(IIdentity identity)
+        public Task<IDeviceListener> GetDeviceListenerAsync(IIdentity identity, Option<string> modelId)
```

Comment:
if this modelId is pnp-domain specific, i think it is better to name it as pnpModelId